proto/internal.go

```diff
@@ -82,3 +83,23 @@ func (m *RaftMessageRequest) GetUser() string {
 	// TODO(marc): we should use security.NodeUser here, but we need to break cycles first.
 	return "node"
 }
+
+// NewRaftMessageRequest returns a new RaftMessageRequest.
+func NewRaftMessageRequest(groupID RangeID, msg raftpb.Message) (*RaftMessageRequest, error) {
+	m := &RaftMessageRequest{GroupID: groupID}
+	var err error
+	m.Msg, err = msg.Marshal()
+	if err != nil {
+		return nil, err
+	}
+	return m, nil
+}
+
+// UnmarshalMsg unmarshals the raft payload.
+func (m *RaftMessageRequest) UnmarshalMsg() (*raftpb.Message, error) {
+	var msg raftpb.Message
```

Comment:
Ah, I see. The code was gone with 65f6231d67b7f73203ff56b5794246007de1c0ff . `RaftMessageRequest.Msg` is converted from `bytes` to `raftpb.Message`.